loopin_test.go

```diff
@@ -315,6 +320,9 @@ func testLoopInTimeout(t *testing.T, externalValue int64) {
 	require.NoError(t, err)
 	cost.Onchain += fee
 
+	// TODO(hieblmi) remove Sleep and fix flake
```

Comment:
I am not clear on why this Sleep here fixes the test flake in Github CI. Locally it does not appear.